src/Datadog.Trace/AppSec/Waf/NativeBindings/Native.cs

```diff
@@ -39,43 +57,200 @@ internal class Native
         [DllImport(DllName)]
         internal static extern IntPtr pw_initAdditiveH(IntPtr powerwafHandle);
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWRet pw_runAdditive(IntPtr context, PWArgs newArgs, ulong timeLeftInUs);
 
         [DllImport(DllName)]
         internal static extern void pw_clearAdditive(IntPtr context);
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWArgs pw_getInvalid();
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWArgs pw_createStringWithLength(string s, ulong length);
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWArgs pw_createString(string s);
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWArgs pw_createInt(long value);
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWArgs pw_createUint(ulong value);
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWArgs pw_createArray();
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern PWArgs pw_createMap();
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern bool pw_addArray(ref PWArgs array, PWArgs entry);
 
         // Setting entryNameLength to 0 will result in the entryName length being re-computed with strlen
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern bool pw_addMap(ref PWArgs map, string entryName, ulong entryNameLength, PWArgs entry);
 
-        [DllImport("Datadog.Trace.ClrProfiler.Native")]
+        [DllImport(DllName)]
         internal static extern void pw_freeArg(ref PWArgs input);
 
 #pragma warning restore SA1300 // Element should begin with upper-case letter
+
+        private static List<string> GetDataDogNativeFolders(string runtimeId)
```

Comment:
It's `Datadog`, not `DataDog` 😉 

(Yes, it's https://github.com/DataDog on GitHub, but nobody dares change it because it could break case sensitive package managers.)

Reply:
If only we'd written it in case insensitive VB.NET!